Remove overlapping legends from radar chart

ayush1298 opened the pull request:
This PR is related to the issue #2156 to remove overlapping legends from the radar chart


### Code Quality
<!-- Please do not delete this -->
- [x] **Code Formatted**: Format the code using `make lint` to maintain consistent style.

The Radar Plot now looks like below where the legend is shifted to the right.
![newplot (4)](https://github.com/user-attachments/assets/7d58a1b8-9fed-4192-be2b-fd01d4f9adcf)

isaac-chung commented:
@ayush1298 thanks for working on this! Could we make it such that it's fully in the margin and not blocking the graph at all?

ayush1298 commented:
> @ayush1298 thanks for working on this! Could we make it such that it's fully in the margin and not blocking the graph at all?
>

@isaac-chung, I mistakenly didn't see that some part of the graph was still blocking previosuly. I have corrected it and added the updated image in the description. Can you have a look at it now?

isaac-chung commented:
@x-tabdeveloping I wonder if the original placement at the bottom worked better? I do like the vertical orientation.

ayush1298 commented:
@x-tabdeveloping, can you have a look at this?

KennethEnevoldsen commented:
@x-tabdeveloping is at a conference, but it sounds from @isaac-chung that we still have issues with overlap? (the picture seems fine to me)

> I wonder if the original placement at the bottom worked better?

From the issue https://github.com/embeddings-benchmark/mteb/issues/2156 it made it very hard to know which model was which.

It might be possible to do a vertical list at the buttom, but I would prefer this solution (assuming the overlap isn't a problem)

x-tabdeveloping commented:
hmm it takes a lot of space up this way though, can you play around with it a bit more to make it look a bit more balanced?
I think the horizontal positioning would also be nice if we could get it to break into multiple lines

ayush1298 commented:
@KennethEnevoldsen @x-tabdeveloping , does the following plot look good? Tried many things, but none of them worked. Also, there is no option to break legend into multiple lines. If this looks good, then I will push a commit.

![newplot (3)](https://github.com/user-attachments/assets/e1d4db64-d425-42a8-8c7a-c987f267234f)

ayush1298 commented:
@isaac-chung I have pushed the commit and added an update plot in the description of the PR.

isaac-chung reviewed (approved):
Great, thank you!
